.editorconfig

```diff
@@ -205,3 +205,7 @@ csharp_space_between_square_brackets = false
 csharp_prefer_braces = true:silent
 csharp_preserve_single_line_blocks = true
 csharp_preserve_single_line_statements = true
+
+# error RS0037: PublicAPI.txt is missing '#nullable enable'
```

Comment:
Will this be done as a separate PR or with further commits in this PR?

Reply:
Each project needs to decide whether they want to enable this. But they cannot enable this right now, because of the issue above (two versions of analyzers involved).